app/components/breadcrumb_nav_component.rb

```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: false
```

Comment:
this was switched because sorbet does not recognize `truncate` as being defined in activesupport (it complains its not defined in the component:

```
app/components/breadcrumb_nav_component.rb:18: Method truncate does not exist on BreadcrumbNavComponent https://srb.help/7003
    18 |    truncate(full_title(breadcrumb), length: 150, separator: ' ')
```

not sure how to deal with that